### PR TITLE
Add page with form install code

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/nls/en-us/dictionary.js [hunk 1]
```diff
@@ -1735,5 +1736,10 @@ define(function(require) {
         'Все в порядке': 'Everything is fine',
         'Вы можете вернуться и отредактировать письмо в любое время': 'You can go back and edit the letter at any time',
         'Сохранить письмо': 'Save letter',
+        'Установка': 'Install',
+        'Код для установки формы': 'Code to install form',
+        'Для того, чтобы подписчики увидели вашу форму, просто добавьте этот код на нужные страницы вашего сайта один раз. Любые изменения, которые вы будете делать в форме, будут появляться на вашем сайте автоматически.': 'To subscribers see your form, simply add the following code into the appropriate pages of your site once. Any changes that you make in a form that will appear on your site automatically.',
+        'Скопировать код': 'Copy code',
+        'Код формы скопирован в буфер обмена': 'Form code copied to clipboard'
```
Comment (rosendi) [inline src/nls/en-us/dictionary.js]: The form code has been copied to clipboard

File: src/nls/en-us/dictionary.js [hunk 2]
```diff
@@ -1735,5 +1736,10 @@ define(function(require) {
         'Все в порядке': 'Everything is fine',
         'Вы можете вернуться и отредактировать письмо в любое время': 'You can go back and edit the letter at any time',
         'Сохранить письмо': 'Save letter',
+        'Установка': 'Install',
```
Comment (rosendi) [inline src/nls/en-us/dictionary.js]: Embed with iFrame

Comment (rosendi) [inline src/nls/en-us/dictionary.js]: or just Embed

File: src/nls/en-us/dictionary.js [hunk 3]
```diff
@@ -1735,5 +1736,10 @@ define(function(require) {
...
+        'Код для установки формы': 'Code to install form',
+        'Для того, чтобы подписчики увидели вашу форму, просто добавьте этот код на нужные страницы вашего сайта один раз. Любые изменения, которые вы будете делать в форме, будут появляться на вашем сайте автоматически.': 'To subscribers see your form, simply add the following code into the appropriate pages of your site once. Any changes that you make in a form that will appear on your site automatically.',
```
Comment (rosendi) [inline src/nls/en-us/dictionary.js]: `Use this embed code to place your form into your pages.`

 I'm working on the second sentence...

File: src/nls/en-us/dictionary.js (hunk 1 above)
Comment (Radiansz) [inline src/nls/en-us/dictionary.js]: "Any changes that you make in a form that will appear on your site automatically."
Second "that" is not needed

Comment (zouxuoz) [inline src/nls/en-us/dictionary.js]: fixed

File: src/nls/en-us/dictionary.js (hunk 3 above)
Comment (zouxuoz) [inline src/nls/en-us/dictionary.js]: fixed

**Comment by Radiansz** [comment]
lgfm

**Pull request merged**